compiler/src/iree/compiler/Codegen/Common/GPU/VectorLayoutProvider.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Codegen/Common/GPU/VectorLayoutProvider.h"
+#include "iree-dialects/Dialect/VectorExt/IR/VectorExtDialect.h"
+
+namespace mlir::iree_compiler {
+
+using namespace IREE::VectorExt;
+
+/// The order in which the dimensions are considered after distribution.
+const LayoutDimension simtLabels[] = {
+    LayoutDimension::BATCHY, LayoutDimension::BATCHX, LayoutDimension::VECTORZ,
+    LayoutDimension::VECTORY, LayoutDimension::VECTORX};
```

Comment:
I realize this is an existing enum, but I'm surprised there's no underscore before the name and the dim letter, e.g., `BATCH_Y`.

Reply:
Right. I wanted to change this layout in the future anyway, so I'm hoping this isn't long lived.